metricbeat/module/coredns/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: coredns
+  title: "coredns"
+  description: >
+    experimental[]
```

Comment:
Let's make this directly beta. To do this remove this line and add `release: beta` as an entry. Same needs to be done on the metricset level.

Was the above automatically generated? If yes, seems like we need to update our generator to also use the `release:` key.

Reply:
Yes @ruflin, this was auto-generated.

Reply:
Updating this :slightly_smiling_face:  https://github.com/elastic/beats/pull/10657